server/starDatabase.sql

```diff
@@ -0,0 +1,23 @@
+
+
+CREATE TABLE stars (
+  id UUID PRIMARY KEY DEFAULT uuid_generate_v4(),
+  name VARCHAR(255) NOT NULL,
+  description TEXT,
+  user_id SERIAL,
```

Comment:
SERIAL is used to ask PG to generate the value each time, which will be an INT value.
That's fine for the Primary Key, but not for Foreign Keys as we need to supply the value.
SO you need to change this to user_id INT